general/management/commands/create_front_page_caches.py

```diff
@@ -33,10 +35,15 @@ def handle(self, **options):
         logger.info("Updating front page caches")
 
         rss_url = settings.FREESOUND_RSS
-        pledgie_campaign = settings.PLEDGIE_CAMPAIGN
-        
+        donations_goal = settings.DONATIONS_GOAL
+        donations_date = settings.DONATIONS_DATE
+
         rss_cache = render_to_string('rss_cache.html', locals())
         cache.set("rss_cache", rss_cache, 2592000) # 30 days cache
 
-        pledgie_cache = render_to_string('pledgie_cache.html', locals())
-        cache.set("pledgie_cache", pledgie_cache, 2592000) # 30 days cache
\ No newline at end of file
+        donations = accounts.models.Donation.objects\
+                .filter(created__gt=donations_date).all().aggregate(Sum('amount'))
```

Comment:
I agree that it will be more flexible if we have a DonationCampaign object. We didn't think of it in the first place to keep it simple, but it would not complicate things too much and would be useful. We could then have campaigns about very specific things also if we want.